docs/faq/questions/using-cypress-faq.mdx

```diff
@@ -1157,9 +1157,7 @@ might be possible by
 
 For end-to-end testing, yes, absolutely. A good example of a fully tested React
 application is our
-[Cypress RealWorld App](https://github.com/cypress-io/cypress-example-realworld)
-and
-[TodoMVC Redux App](https://github.com/cypress-io/cypress-example-todomvc-redux).
+[Cypress RealWorld App](https://github.com/cypress-io/cypress-example-realworld).
```

Comment:
is this supposed to be `cypress-realworld-app`? `cypress-example-realworld` redirects to `cypress-example-conduit-app` which has been archived.

```suggestion
[Cypress RealWorld App](https://github.com/cypress-io/cypress-realworld-app).
```

Reply:
Weird